pints/_log_likelihoods.py

```diff
@@ -237,6 +237,228 @@ def __call__(self, x):
         )
 
 
+class CombinedGaussianLogLikelihood(pints.ProblemLogLikelihood):
```

Comment:
"Combined gaussian" sounds much more broad than what this actually is. For example it doesn't combine two N(mu, sigma) log likelihoods